src/index.ts

```diff
@@ -22,6 +22,7 @@ import type { CucumberJsJsonReporterInterface, AttachmentType, CucumberAttachmen
 import type { HookStatsExtended, SuiteStatsExtended, TestStatsExtended } from './types/wdio.js'
 
 const log = logger('wdio-cucumberjs-json-reporter')
+const { existsSync, outputJsonSync, readJsonSync } = fsExtra
```

Comment:
Let's avoid having extra assignment and just use e.g. `fs.outputJsonSync`

Reply:
Thank you, updated